src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/profiler/ProfilerEditingTarget.java

```diff
@@ -47,23 +55,26 @@
 
 public class ProfilerEditingTarget implements EditingTarget
 {
-   public static final String PATH = "profiler://";
-   
    @Inject
-   public ProfilerEditingTarget(ProfilerPresenter presenter,
-                                Commands commands,
-                                EventBus events)
+   public ProfilerEditingTarget(
+      ProfilerPresenter presenter,
+      Commands commands,
+      EventBus events,
+      ProfilerServerOperations server,
+      GlobalDisplay globalDisplay)
    {
       presenter_ = presenter;
       commands_ = commands;
       events_ = events;
+      server_ = server;
+      globalDisplay_ = globalDisplay;
    }
 
    public String getId()
    {
       return doc_.getId();
    }
-   
```

Comment:
Try to avoid whitespace-only changes (e.g. disable 'strip trailing whitespace' in your editor save prefs if possible)